README.md

```diff
@@ -7,4 +7,50 @@ Course in Ecology and Evolutionary Biology
 [![Build Status](https://travis-ci.org/UofTCoders/rcourse.svg?branch=master)](https://travis-ci.org/UofTCoders/rcourse)
 [![DOI](https://zenodo.org/badge/97400494.svg)](https://zenodo.org/badge/latestdoi/97400494)
 
+
+## Description
+
+Quantitative Methods in R for Biology is a course aimed at undergraduates at a third year level or above. 
+The course covers statistics and data analysis for ecology and reproducible quantitative methods in R. 
+
+Statistical analysis, modelling, simulation, and data analysis are essential skills for applying 
+ecology concepts to data. This course is designed to meet a growing demand for reproducible, openly accessible, 
+analytically thorough, and well documented science. Students will learn to develop ecological population models, 
+analyze data, and document their research using the R programming language. No prior programming experience is required.
+
 For more detail on the course, check out the [syllabus](https://uoftcoders.github.io/rcourse/).
+
+## Instructional Design
+
+The lectures in this course are designed to be presented using a participatory live-coding approach.
+This involves an instructor typing and running code in [RStudio](https://www.rstudio.com/) in front of the class, while the class follows
+along using their own computers. Challenges are interspersed in the lesson material, allowing students to 
+collaboratively work on smaller coding problems for a few minutes. All lesson materials are provided ahead
+of time on the course website for students to refer to during lectures.
+
+The bulk of the course's assessment structure involves weekly assignments. These assignments
+are primarily code-based, and are designed to also be completed in RStudio using the R Markdown format.
+
+At the end of the course, students undertake a group project, wherein they attempt to address a scientific
+question by applying techniques learned over the course to open ecological data. At the end of the semester,
+groups present their work in a conference-style presentation, and submit a report in the style of a scientific paper. 
+
+## Lecture Content
+
+The course's lesson material is broadly subdivided into three main topics:
```

Comment:
s/three/four 

although Sara likes 'three' as a nice round number, so maybe we could combine 2 and 3 here like you had before?